plane/source/docs/tuning-cruise.rst

```diff
@@ -1,37 +1,43 @@
 
 In the automatic throttle controlled modes, :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>` is used for the target airspeed if an airspeed sensor is being used, while :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` will be set for the throttle value if no sensor is used. The :ref:`THROTTLE_NUDGE<THROTTLE_NUDGE>` option allows the pilot to tweak this value while in flight with the throttle, if desired in these modes.
 
-.. note:: While :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` is not used when using an airspeed sensor directly, it is important to set it at a working value, since it will be used in case of an airspeed sensor failure.
+.. warning:: While :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` is not used when using an airspeed sensor directly, it is important to set it at a working value, since it will be used in case of an airspeed sensor failure.
 
-AHRS LEVEL Attitude
+AHRS Level Attitude
 ===================
 
-During ACC Calibration, a "level" attitude position is set. Usually this is set to 2-6 degrees positive by blocking up the plane appropriately, such that the plane will have its wings at that small angle ("angle of attack" or AOA) to produce lift at the cruising speed to hold altitude.
+During :ref:`common-accelerometer-calibration`, a "level" attitude position is set with the wings perfectly level. Often for planes, the trim "level" position has the nose raised a few degrees. This is explained in the diagrams below and is known as the trim condition. Most planes fly with a few degrees of Angle of Attack (AOA) and while many planes will have some Angle of Incidence (i.e., the cord of the wing is at a positive angle to the fuselage cord) built-in, some do not, and some need a slightly higher AOA to fly at lower cruise speeds. If the level step of calibration is done with the plane's fuselage line level, initial flights will be safe, but the aircraft may lose altitude due to not having enough lift. There are a few options to produce more lift and improve the trim condition for FBWA mode:
 
-This may not be what actually results in the above modes when cruising since the autopilot's priority to obtain the correct combination of pitch and throttle and/or speed to maintain level altitude flight.
+- Add a few extra degrees nose up when during the Level step of 3-axis calibration.
+- Add a few extra degrees nose up and use the  ``Calibrate Level`` button on the Mission Planner page to adjust the AHRS_TRIM parameters. AHRS_TRIM parameters can only change the difference between the autopilot's plane and "level" by 10 degrees maximum. If more is needed, (e.g., the autopilot is mounted slightly downward), then you can use :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` to alter the AOA manually.
+- Add a few degrees nose up to :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>`. This will also add an offset to the artificial horizon on a GCS or an OSD.
```

Comment:
```suggestion
- **(Preferred)** Add a few degrees nose up to :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>`. This will also add an offset to the artificial horizon on a GCS or an OSD.
```